EventEnabledInsurance/QuoteLifecycleSimulator/deploy-simulator.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+#******************************************************************************
+# Licensed Materials - Property of IBM
+# (c) Copyright IBM Corporation 2019. All Rights Reserved.
+#
+# Note to U.S. Government Users Restricted Rights:
+# Use, duplication or disclosure restricted by GSA ADP Schedule
+# Contract with IBM Corp.
+#******************************************************************************
+
+# PREREQUISITES:
+#   - Logged into cluster on the OC CLI (https://docs.openshift.com/container-platform/4.4/cli_reference/openshift_cli/getting-started-cli.html)
+#
+# PARAMETERS:
+#   -n : <namespace> (string), Defaults to 'cp4i'
+#   -p : <PG_PORT> (string), Defaults to '5432'
+#   -t : <TICK_MILLIS> (string), Defaults to '1000'
+#   -m : <MOBILE_TEST_ROWS> (string), Defaults to '10'
+#
+#   With defaults values
+#     ./prereqs.sh
+#
+#   With overridden values
+#     ./prereqs.sh -n <namespace> -p <PG_PORT> -t <TICK_MILLIS> -m <MOBILE_TEST_ROWS>
+
+function usage() {
+  echo "Usage: $0 -n <namespace> -p <PG_PORT> -t <TICK_MILLIS> -m <MOBILE_TEST_ROWS>"
+  exit 1
+}
+
+namespace="cp4i"
+tick="\xE2\x9C\x85"
+cross="\xE2\x9D\x8C"
+all_done="\xF0\x9F\x92\xAF"
+SUFFIX="eei"
+POSTGRES_NAMESPACE="postgres"
+ACE_CONFIGURATION_NAME="ace-policyproject-$SUFFIX"
+PG_PORT=5432
+TICK_MILLIS=1000
+MOBILE_TEST_ROWS=10
+
+while getopts "n:p:t:m:" opt; do
+  case ${opt} in
+  n)
+    namespace="$OPTARG"
+    ;;
+  p)
+    PG_PORT="$OPTARG"
+    ;;
+  t)
+    TICK_MILLIS="$OPTARG"
+    ;;
+  m)
+    MOBILE_TEST_ROWS="$OPTARG"
+    ;;
+  \?)
+    usage
+    exit
+    ;;
+  esac
+done
+
+CURRENT_DIR=$(dirname $0)
+echo "INFO: Current directory: '$CURRENT_DIR'"
+echo "INFO: Namespace: '$namespace'"
+echo "INFO: Suffix for the postgres is: '$SUFFIX'"
+
+PG_PASSWORD=$(echo $(oc get secret -n $namespace postgres-credential-eei -o json | jq -r '.data.password') | base64 --decode)
+
+if [[ -z "${namespace// }" || -z "${PG_PORT// }" || -z "${TICK_MILLIS// }" || -z "${MOBILE_TEST_ROWS// }" || -z "${PG_PASSWORD// }" ]]; then
+  echo -e "$cross ERROR: Some or all mandatory parameters are empty"
+  usage
+fi
+
+echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
+
+oc project $namespace
+
+echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
+
+echo -e "INFO: Getting postgres pod name and service name for the simulator app host"
+DB_POD=$(oc get pod -n $POSTGRES_NAMESPACE -l name=postgresql -o jsonpath='{.items[].metadata.name}')
+DB_SVC="$(oc get cm -n $POSTGRES_NAMESPACE postgres-config -o json | jq '.data["postgres.env"] | split("\n  ")' | grep DATABASE_SERVICE_NAME | cut -d "=" -f 2- | tr -dc '[a-z0-9-]\n').postgres.svc.cluster.local"
```

Comment:
```suggestion
DB_SVC="postgresql.$POSTGRES_NAMESPACE.svc.cluster.local"
```
I don't see the point in making this so complicated to find the service name, and it's not using the var for the postgres namespace. I suggest changing to this (I've made this change to the DDD prereqs on my branch).